README.md

```diff
@@ -16,6 +22,101 @@ The following platforms are officially supported (tested):
 - **Operating System:** Ubuntu Linux 20.04
 - **Architectures:** amd64, arm64
 
+## Usage
+
+### Installation
+
+You can install the Frequenz Electricity Trading API client via pip. Replace `VERSION` with the specific version you wish to install.
+
+```
+# Choose the version you want to install
+VERSION=0.1.0
+pip install frequenz-client-electricity-trading==$VERSION
+```
+
+### Initialization
+
+First, initialize the client with the appropriate server URL and API key.
+```python
+from frequenz.client.electricity_trading import Client
+
+# Change server address if needed
+SERVICE_URL = "electricity-trading-testing.api.frequenz.com:443"
```

Comment:
we will need to set the URL to use the latest version of the client as follows:
```suggestion
SERVICE_URL = "grpc://electricity-trading-testing.api.frequenz.com:443?ssl=true"
```

Reply:
Would it make sense to have it in 2 steps like so or rather not?
```
SERVICE_ADDRESS = "electricity-trading.api.frequenz.com:443"
SERVICE_URL = f"grpc://{SERVICE_ADDRESS}?ssl=true"
```

Reply:
I personally prefer 1 step/line as the address will be only used in the URL. Though I don't have any strong opinion about doing it in 2 steps

Reply:
makes sense, I just thought that in the future the `SERVICE_ADDRESS` might be in the config in the app, while `SERVICE_URL` is more implementation specific but actually I guess we can just have the URL in the config